### DISCUSSION TITLE
Tunturi E60 Performance combatibly

### DISCUSSION THREAD

**Discussion by zteady** [cagnulein/qdomyos-zwift#920] [Q&A]
Hello!

Is Tunturi E60 supported? How i would know if it support auto resistance?

**Comment by cagnulein** [comment]
hi @zteady send me a nrfconnect screenshot (free app on the store) and i will let you know

**Comment by zteady** [comment]
Can you specify what part of information i should screenshot?

Thank you!

**Reply by cagnulein** [reply]
@zteady open nrfconnect and send me a screenshot showing all the devices that it found

**Reply by zteady** [reply]
![E51109BA-014D-4183-B33B-D2A039A9BC8A](https://user-images.githubusercontent.com/112695928/190879233-7379dee6-5d34-4858-a04b-f1230cf8f19a.png)
![33516C80-3F73-43D8-8643-FEFF26D3487B](https://user-images.githubusercontent.com/112695928/190879239-d7d2f813-9492-4813-b551-d551ee9194ef.png)
![5D4DDCB4-99C5-4BF2-B79F-2F10E6473C14](https://user-images.githubusercontent.com/112695928/190879241-caf4b4cb-5ca3-4b75-bf33-729c0e1ff7ee.png)

**Reply by cagnulein** [reply]
hi @zteady i guess i already implemented this in the beta

send me an email to roberto.viola83 at gmail.com indicating this ticket and if you are on android or iOS. Thanks

**Reply by cagnulein** [reply]
last question @zteady 
is this a bike or a treadmill?

**Comment by zteady** [comment]
Hello! Its a bike.

It wont connect app at all.

I actually saw in here that you have implemented Tunturi Fitcycle 50i but
that user had connection first.

I will sen this to your email too.

Thank you!
